content/posts/newsletter-004/index.md

```diff
@@ -46,7 +48,7 @@ If needed, a section can be split into subsections with a "------" delimiter.
 
 ### [Le Train Dispatcher][ltd] - Route trains in a simulated rail network
 
-![Demo of Le Train Dispatcher](/assets/newsletter-004/ltd.gif)
+![Demo of Le Train Dispatcher](ltd.gif)
```

Comment:
The old file at /assets/newsletter-004/ltd.gif should be removed too

Reply:
Whoops, I did but it got brough back by my rebase 😅 Fixed!